buildozer/__init__.py

```diff
@@ -879,10 +871,27 @@ def namify(self, name):
     def root_dir(self):
         return realpath(dirname(self.specfilename))
 
+    @property
+    def user_build_dir(self):
+        '''The user-provided build dir, if any.'''
```

Comment:
I like that we now have a dedicated method for this. But I also have a nitpick.
PEP-0257 say docstring should be double quotes, not single. Same in other docstrings. I know buildozer and p4a is already inconsistent here.
https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring